app/views/users/invitations/new.html.erb

```diff
@@ -3,6 +3,8 @@
 <%= form_for resource, :as => resource_name, :url => invitation_path(resource_name), :html => {:method => :post} do |f| %>
   <%= devise_error_messages! %>
 
+  <%= f.hidden_field :organisation_id %>
```

Comment:
Do we need this?
Given we are newing up the User with the organisation id already assigned.

Just worried that having this in the markup opens up the possibility of someone modifying it.

Reply:
Thats a valid concern. We could assign it in the create action, but its not as clean. It would require either implementing a large part of the controller from the gem https://github.com/scambra/devise_invitable/blob/master/app/controllers/devise/invitations_controller.rb#L18

We could do it as a before create action and manually update the params, then call super.... Let me see.